src/providers/push/apns/types.ts

```diff
@@ -20,12 +20,10 @@ export interface ApnsConfigType {
 
 export interface ApnsMessage extends PushMessage {
   receiver: ApnsReceiver;
-  // sonarignore:start
   options: {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    [key: string]: any;
+    [key: string]: any; //NOSONAR
   };
```

Comment:
Can use the already provided type in loopback: `AnyObject`

Reply:
done